src/pages/install/linux.md

```diff
@@ -0,0 +1,24 @@
+
+# Installing RGBDS on Linux
+
+First, please check if Linux is available in your distribution's software center or package manager repositories.
+If a satisfactory version is available, it is a good idea to install from there.
+
+If not, and you want to install version v0.6.1 or newer, you can download our pre-built binaries.
+
+1. Select [the version you want](/docs/).
```

Comment:
```suggestion
1. Select [the version you want](/docs/) and follow its link.
```

Reply:
```suggestion
1. Click the link of the version you want [from the list](/docs/).
```

Reply:
"the link to the version", but otherwise yes. (And likewise for other instances.)